R/tm_a_pca.R

```diff
@@ -24,24 +24,25 @@
 #'
 #' # ADSL example
 #'
-#' library(nestcolor)
-#' ADSL <- teal.modules.general::rADSL
-#'
+#' data <- teal_data()
+#' data <- within(data, {
+#'   library(nestcolor)
+#'   ADSL <- teal.modules.general::rADSL
+#' })
+#' datanames <- c("ADSL")
+#' datanames(data) <- datanames
+#' join_keys(data) <- cdisc_join_keys(!!!datanames)
```

Comment:
```suggestion
#' join_keys(data) <- default_cdisc_join_keys[datanames]
```

Reply:
changed in tmhermos as well https://github.com/insightsengineering/teal.modules.hermes/pull/342/commits/ba3eb957c3903928f19d1ae6b5ab09c162ebc866